src/Provider/Provider.Web/ViewModels/VacancyPreview/VacancyPreviewViewModel.cs

```diff
@@ -32,7 +34,10 @@ public class VacancyPreviewViewModel : DisplayVacancyViewModel
         public VacancyPreviewSectionState WageTextSectionState { get; internal set; }
         public VacancyPreviewSectionState DescriptionsSectionState { get; internal set; }
         public VacancyPreviewSectionState WorkingWeekSectionState { get; internal set; }
-        
+
+        public EntityValidationResult SoftValidationErrors { get; internal set; }
+        public bool HideValidationSummary { get; internal set; }
```

Comment:
Can this be called `CanHideValidationSummary` please

Reply:
👍